lang/en/common.php

```diff
@@ -130,6 +130,8 @@
     'empty_transactions' => "You don't have any transactions yet. Once transactions have been made, they will show up here.",
     'error' => 'Error',
     'refresh_metadata' => 'Refresh Metadata',
+    'refresh_collection_title' => 'Refresh your collection',
+    'refresh_collection_description' => 'You can refresh data every 15 minutes.',
```

Comment:
remove the refresh related UI as we don't have that feature implemented yet and it will otherwise block the PR